src/app/components/json.tsx

```diff
@@ -21,6 +21,7 @@ function parseJsonReadable(value: any) {
 export function Json(value: any) {
   return (
     <Box
+      maxWidth={{ base: '70vw', md: 'pageWidth' }}
```

Comment:
Where does this 70% value come from?

IMO this seems like something that belongs on a wrapper component, as we might want to use this JSON component again in another context where we don't want these rules.

Reply:
Good point. I will move it up to a wrapper if needed. 

I chose it by testing at the small breakpoint and picking the best fit. 
https://github.com/user-attachments/assets/23bba347-52b3-48fb-be1c-ffd9abda5b62

I've doing some more adjustments higher up so it may not even be needed. 

I'll move this to DRAFT as its a WIP and can re-open when ready for re-review